src/contexts/ScrollContext.tsx

```diff
@@ -16,26 +16,31 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { createContext, ReactNode, useContext } from 'react'
+import { createContext, MutableRefObject, ReactNode, RefObject, useContext, useRef } from 'react'
+import { FlatList, ScrollView } from 'react-native'
 import { SharedValue, useSharedValue } from 'react-native-reanimated'
 
+type ScrollableView = ScrollView | FlatList
+
+export type ScrollDirection = 'up' | 'down' | null
```

Comment:
Should we get rid of this since we are not using it and clean up this component, or leave it for potential future use?

Reply:
We may need to use it soon and it's not too much noise. Let's keep it for a few iterations? If we see then that we don't need it, we'll remove it.